libs/server/repository/core/domain/src/lib/utils/apply-type.util.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  RepositoryCommonSettingDataType,
+  RepositoryCommonSettingValueDataType
+} from '@pimp-my-pr/shared/domain';
+
+export const applyType = (
```

Comment:
applyCommonSettingValueType

Reply:
but still I'm not sure if it is a part of the domain

Reply:
Where, do You suggest, to put this util function?